File: Source/ArrayRelatedOperations/print.js
function printArray(a, size) {
    var output = '';
    for(var i = 0; i < size; i++) {
        output += a[i] + ' ';
    }
    console.log(output);
}

var a = [1, 2, 3, 4, 5];
var size = a.length;

printArray(a, size);

File: Source/ArrayRelatedOperations/print.c
#include <stdio.h>

void printArray(int a[], int size) {
    int i;
    for(i = 0; i < size; i++) {
        printf("%d ", a[i]);
    }
}

int main() {
    int a[] = {1, 2, 3, 4, 5};
    int size = sizeof(a)/sizeof(a[0]);

    printArray(a, size);

    return 0;
}
